locales/en/kafkaoverview-v2.json

```diff
@@ -0,0 +1,28 @@
+{
+  "cloudProvidersTitle": "Cloud providers",
+  "contactSalesCardCallToActionButton": "Contact sales",
+  "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription.",
+  "contactSalesCardTitle": "Contact sales",
+  "dataTransfer": "Data transfer",
+  "firstStreamingUnit": "1 streaming unit",
+  "heroCallToActionButton": "",
+  "heroDescription": "Red Hat OpenShift Streams for Apache Kafka is a managed cloud service that provides a streamlined developer experience for building, deploying, and scaling real-time applications in hybrid-cloud environments. The combination of seamless operations across distributed microservices, large data transfer volumes, and managed operations allows teams to focus on core competencies, accelerate application velocity and reduce operational cost.",
+  "heroTagline": "Fully hosted and managed service for stream-based applications",
+  "heroTitle": "Get started with Red Hat OpenShift Streams for Apache Kafka",
+  "heroTryItButton": "",
+  "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity.",
+  "kafkaInstanceCapacityfooter1": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact",
+  "kafkaInstanceCapacityfooter2": "to discuss your options. You can check your current storage usage by",
+  "kafkaInstanceCapacityTitle": "Kafka instance capacity",
+  "microsoftAzureDescription": "Coming soon",
+  "pricingModalTitle": "Pricing model",
+  "purchaseCardCallToActionButton": "Purchase now",
+  "purchaseCardMainText": "In the United States or Canada, purchase a pay-as-you-go subscription for up to 25 streaming units through Red Hat Marketplace.",
+  "purchaseCardTitle": "Purchase now",
+  "secondStreamingUnit": "2 streaming unit",
```

Comment:
A couple of notes about this string:
1. There should be an "s" at the end: "2 streaming units"
2. This string used to display in a table header for the Kafka instance capacity card, but it's missing from the current implementation.

Here is an example of what this row looked like in a previous iteration of this:
![image](https://user-images.githubusercontent.com/21063328/166522144-73e247bc-6a4e-4a88-a250-e55ddebf5a78.png)